src/test/java/book_store/controller/ShoppingCartControllerTest.java

```diff
@@ -0,0 +1,130 @@
+package book_store.controller;
+
+import book_store.dto.cartItemDto.CartItemDto;
+import book_store.service.ShoppingCartService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.authority.AuthorityUtils;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+import book_store.dto.shoppingCart.AddToCartRequest;
+import book_store.dto.shoppingCart.ShoppingCartDto;
+import book_store.dto.cartItemDto.UpdateCartItemRequest;
+import book_store.model.User;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import java.util.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.authentication;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@WithMockUser(username = "user", roles = {"USER"})
+class ShoppingCartControllerTest {
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private ShoppingCartService shoppingCartService;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Test
+    @DisplayName("Get shopping cart")
+    void getShoppingCart() throws Exception {
```

Comment:
test name invalid, should be in pattern like:
`<methodUnderTest>_<state>_<expectedBehavior>`